docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+---
+title: "Project property settings in Visual Studio"
+description: "Describes the Visual Studio project properties available to a .NET MAUI app. The properties and settings are related to building the app, configuring debug options, and other settings related to building the application for Windows, Android, and iOS."
+ms.date: 07/11/2022
+---
+
+# Project configuration in .NET MAUI
+
+.NET MAUI uses a [single-project system](../fundamentals/single-project.md) to manage most of your multi-platform app. This is different from .NET MAUI's predecessor, Xamarin, which used multiple projects, one for each platform. Project configuration in .NET MAUI is similar to other projects in Visual Studio, right-click on the project in the **Solution Explorer**, and select **Properties**.
```

Comment:
Is the sentence about Xamarin adding anything useful? Xamarin users will know that. .NET MAUI users don't need to know that?